packages/forms/src/directives/validators.ts

```diff
@@ -420,17 +417,11 @@ export class RequiredValidator implements Validator {
   selector:
       'input[type=checkbox][required][formControlName],input[type=checkbox][required][formControl],input[type=checkbox][required][ngModel]',
   providers: [CHECKBOX_REQUIRED_VALIDATOR],
-  host: {'[attr.required]': 'required ? "" : null'}
+  host: {'[attr.required]': 'enabled() ? "" : null'}
```

Comment:
Similar to the `RequiredValidator`, we should use `_enabled` here (instead of `enabled()`):
```suggestion
  host: {'[attr.required]': '_enabled ? "" : null'}
```